.github/workflows/inference_unittests.yaml

```diff
@@ -0,0 +1,32 @@
+name: Unit Test for Inference
+
+on:
+  push:
+    branches:
+      - md-unittests
+    paths: 
+      - chatbot_inference/**
+      
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: test 
+        run: ls chatbot_inference
+        working-directory: ${{ env.GITHUB_WORKSPACE }}
+      - name: Install libs
+        run: |
+          #python -m pip install --upgrade pip
+          pip install -r chatbot_inference/requirements.txt
+        working-directory: ${{ env.GITHUB_WORKSPACE }}
+      - name: Unit test execution
```

Comment:
I was thinking we can use the find -exec functionality to execute unit test files. First, we should come up with a common naming conventions for unit test tile. Something like unittest_*.py. I think you use test_* but lets change that to unittest_*. Then, the command to execute all the unit tests in a directory would be:
.
      - name: Execute unit tests
        run: |
          find ./ncf_inference -name unittest*.py -exec python {} \;
.
This way, any unit test file added to the directory automatically gets appended to the list.